PubMed.js

```diff
@@ -176,6 +179,16 @@ function getSearchResults(doc, checkOnly) {
 			checked: checkbox && checkbox.checked
 		};	
 	}
+
+	for (var i=0; i<docsums.length; i++) {
+		var docsum_title = ZU.xpathText(docsums[i], './/div[@class="docsum-wrap"]/div[@class="docsum-content"]/a[@class="labs-docsum-title"]')
```

Comment:
so you need to construct `items` array similar way as it happens in lines 155-181, only css selectors for docsums are different